cmd/automation/list.go

```diff
@@ -15,9 +15,10 @@ func newListCmd() *cobra.Command {
 		Short: "List available bitswan workspace automations",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			workspaceName, err := config.GetWorkspaceName()
+			configManager := config.NewAutomationServerConfig()
```

Comment:
Shouldn't be named configManager but automationServerConfig or maybe just config